src/main/java/code401challenges/tree/BreadthSearch.java

```diff
@@ -0,0 +1,25 @@
+package code401challenges.tree;
+
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class BreadthSearch {
+    Queue<Node> t = new LinkedList();

```

Comment:
If you have any problems with different types of nodes, try: Queue<Node<T>> t = new LinkedList();